src/_styles/_hero.scss

```diff
@@ -0,0 +1,4 @@
+img {

```

Comment:
Let's namespace this `img` with class, eg. "header-avatar" to avoid overwriting styles for all `img` on page.